specification/assets/gherkin/evaluation.feature

```diff
@@ -1,3 +1,4 @@
+@deprecated
```

Comment:
What is the consequences of this tag for the consumers of the `.feature`? Tried to investigate but I couldn't find any.

Reply:
The new gherkin implementations should autoload all gherkin files in this directory - with @deprecated we add the possibility to filter this test out, as we might need it for sdks not migrated to the new way of testing. - it is a gherkin test filter